products/bash/api.yaml

```diff
@@ -0,0 +1,106 @@
+swagger: '2.0'
```

Comment:
Looks like this file was added by accident?

Reply:
Yes this file needs to be deleted